src/defaults.js

```diff
@@ -1,6 +1,14 @@
 // Global defaults one may want to configure
-export default {
```

Comment:
I’m not opposed to the change, but curious for the rationale?

Reply:
I wasn't 100% sure if `this.verbose` would work when referencing the default export, but it seems to -- I'll revert this.